mapiproxy/dcesrv_mapiproxy.c

```diff
@@ -286,8 +287,6 @@ static void mapiproxy_op_unbind(struct dcesrv_connection_context *context, const
 		talloc_free(private);
 	}
 
-	talloc_free(context);
-
```

Comment:
Sorry but I don't understand this change and how it is related with async notifications. Could you explain it to me?